### PR TITLE
KBS2 - Update sdl2.re for Key.Keycode

### PR DESCRIPTION
**Pull request by K-bNd**
I added some new values to account for numeric pad

### CODE REVIEW

File: src/sdl2.re [hunk 1]
```diff
@@ -297,7 +297,7 @@ module Keycode = {
   let minus = 45;
   let period = 46;
   let slash = 47;
-  let carrot = 94;
```
Comment (K-bNd) [inline src/sdl2.re]: Typo my bad

Comment (Et7f3) [reply src/sdl2.re]: Everyone like :rabbit:.

File: src/sdl2.re [hunk 2]
```diff
@@ -285,12 +285,19 @@ module Keycode = {
...
-
+  let return = 13;
   let escape = 27;
-
+  let space = 32;
+  
+  let left_paren = 40;
+  let right_paren = 41;
+  
+  let multiply = 42;
```
Comment (Et7f3) [inline src/sdl2.re]:
```suggestion
  let asterisk = 42;
```
To respect sdl convention.

File: src/sdl2.re [hunk 3]
```diff
@@ -304,6 +311,27 @@ module Keycode = {
...
+  let p_digit2 = 1073741914;
+  let p_digit3 = 1073741915;
+  let p_digit4 = 1073741916;
+  let p_digit5 = 1073741917;
+  let p_digit6 = 1073741918;
+  let p_digit7 = 1073741919;
+  let p_digit8 = 1073741920;
+  let p_digit9 = 1073741921;
+  
+
```
Comment (Et7f3) [inline src/sdl2.re]:
```suggestion
  let p_digit1 = 1073741913;
  let p_digit2 = 1073741914;
  let p_digit3 = 1073741915;
  let p_digit4 = 1073741916;
  let p_digit5 = 1073741917;
  let p_digit6 = 1073741918;
  let p_digit7 = 1073741919;
  let p_digit8 = 1073741920;
  let p_digit9 = 1073741921;
  let p_digit0 = 1073741922;
```
I have sorted like sdl.

File: src/sdl2.re [hunk 4]
```diff
@@ -304,6 +311,27 @@ module Keycode = {
   let digit7 = 55;
   let digit8 = 56;
   let digit9 = 57;
+  
+  let pad_multiply = 1073741909;  
+  let pad_plus = 1073741911;
+  let pad_minus = 1073741910;
+  let pad_period = 1073741923;
+  let pad_slash = 1073741908;
```
Comment (Et7f3) [inline src/sdl2.re]:
```suggestion
  let pad_divide = 1073741908;
  let pad_multiply = 1073741909;  
  let pad_minus = 1073741910;
  let pad_plus = 1073741911;
  let pad_period = 1073741923;
```
I have sorted like sdl.

**Review by Et7f3 — APPROVED** [review]

File: src/sdl2.re [hunk 5]
```diff
@@ -285,12 +285,19 @@ module Keycode = {
   // https://wiki.libsdl.org/SDLKeycodeLookup
   let unknown = 0;
   let backspace = 8;
-
+  let return = 13;
   let escape = 27;
```
Comment (Et7f3) [inline src/sdl2.re:290]:
```suggestion

  let escape = 27;

```

**Comment by Et7f3** [comment]
thanks @KBS2

**Pull request merged**